Services/AccessControl/classes/class.ilObjectPermissionStatusGUI.php

```diff
@@ -458,8 +458,8 @@ function getAvailableRolesTableData()
 		{
 			$result_set[$counter]["img"] = in_array($role['obj_id'],$this->user_roles) ? self::IMG_OK : self::IMG_NOT_OK;
 
-			if(is_subclass_of($this->object->plugin, 'ilPlugin') && $role["parent"] == $this->object->getRefId()) {
-				$result_set[$counter][] = ilPlugin::lookupTxtById($this->object->getType(), ilObjRole::_removeObjectId($role["title"]));
+			if(is_subclass_of($this->object, 'ilObjPlugin') && $role["parent"] == $this->object->getRefId()) {
```

Comment:
@fwyss-srag how comes the change from `$this->object->plugin` to `$this->object`?

Reply:
At this point it is checked whether the current object is a plugin based on `ilPlugin`, or not? Since `$plugin` is `protected` by `ilObjPlugin`, it thought that this can also be checked by using `object` and `ilObjPlugin`.

Reply:
ok, but there is no such class (at least in trunk), do you mean `ilObjectPlugin`? you can use `ilObjectPlugin::class` instead of the string

Reply:
@chfsx Yes, typo, I mean `ilObjectPlugin', I will fix it